### PR TITLE
Update workshop schedule

### PR DESCRIPTION
**Pull request by mabraham**
Let's finalize this so we can update the ENCCS workshop front page also! @wikfeldt

### CODE REVIEW

File: content/index.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ for recommended prerequisite knowledge.
    30 min ; :doc:`non-blocking-communication-pt2`
    30 min ; :doc:`one-sided-pt1`
```
Comment (mabraham) [inline content/index.rst]: @robertodr What do you plan here and at line 63? Please push a change that suits you!

Comment (robertodr) [reply content/index.rst]: You mean time-wise?

Comment (mabraham) [reply content/index.rst]: Yes, so we have input to fix up https://enccs.se/events/2020/intermediate-topics-in-mpi/

Comment (robertodr) [reply content/index.rst]: 30 minutes each for comms&groups and datatypes. And hopefully 60 and 45 minutes for one-sided part 1 and one-sided part 2, respectively, will be enough.

File: content/collective-communication-pt2.rst [hunk 2]
```diff
@@ -150,165 +151,6 @@ Code-along exercise: scatter and gather
    * Download a :download:`working solution <code/collective-communication-scatter-and-gather-solution.c>`
 
 
-All-gather
```
Comment (robertodr) [inline content/collective-communication-pt2.rst:153]: Is this moving somewhere else? Or is this going away forever?

Comment (mabraham) [reply content/collective-communication-pt2.rst:153]: It's a dupe of content that was incompletely split into pt3

**Pull request merged**